Tribler/community/bartercast4/statistics.py

```diff
@@ -144,7 +144,7 @@ def close(self):
  UNIQUE (type, peer));
 
 CREATE TABLE option(key TEXT PRIMARY KEY, value BLOB);
-INSERT INTO option(key, value) VALUES('database_version', '""" + str(LATEST_VERSION) + """');
+INSERT INTO option(key, value) VALUES('database_version', '""" + str(LATEST_VERSION) + u"""');
```

Comment:
Not really, i just noticed it and included it. Seems a bit overkill to put it in a separate pull request. Can do if you want it ofc...